sql/connect/server/src/main/scala/org/apache/spark/sql/connect/ml/MLUtils.scala

```diff
@@ -228,39 +225,66 @@ private[ml] object MLUtils {
     instance
   }
 
+  /**
+   * Replace the operator with the value provided by the backend.
+   */
+  private def replaceOperator(sessionHolder: SessionHolder, name: String): String = {
+    SparkConnectPluginRegistry
+      .mlBackendRegistry(sessionHolder.session.sessionState.conf)
+      .view
+      .map(p => p.transform(name))
+      .find(_.isPresent) // First come, first served
+      .getOrElse(Optional.of(name))
+      .get()
```

Comment:
nit: Could use `import scala.jdk.OptionConverters._`

```suggestion
      .flatMap(p => p.transform(name).toScala)
      .headOption // First come, first served
      .getOrElse(name)
```

Reply:
Really appreciate @gerashegalov  for the knowledge. But I was following the existing pattern from https://github.com/apache/spark/blob/master/sql/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala#L1620, So I'd like to keep the original one. Probably, you can submit a PR to fix them both.